<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>370</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>680</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>680</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Início</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget {
background-color:  rgb(21, 143, 181);
}</string>
   </property>
   <widget class="QPushButton" name="btnMatricula">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>200</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(255, 123, 28);
border-radius: 15px;
background-color: rgb(255, 123, 28);
color: #fff;</string>
    </property>
    <property name="text">
     <string>Matrícula</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/matricula.png</normaloff>icons/matricula.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRematricula">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>200</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(255, 123, 28);
border-radius: 15px;
background-color: rgb(255, 123, 28);
color: #fff;</string>
    </property>
    <property name="text">
     <string>Rematrícula</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/rematricula.png</normaloff>icons/rematricula.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAlunos">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>200</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(255, 123, 28);
border-radius: 15px;
background-color: rgb(255, 123, 28);
color: #fff;</string>
    </property>
    <property name="text">
     <string>Alunos</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/alunos.png</normaloff>icons/alunos.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMaterias">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>130</y>
      <width>200</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(255, 123, 28);
border-radius: 15px;
background-color: rgb(255, 123, 28);
color: #fff;</string>
    </property>
    <property name="text">
     <string>Matérias</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/materias.png</normaloff>icons/materias.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnProfessores">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>200</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(255, 123, 28);
border-radius: 15px;
background-color: rgb(255, 123, 28);
color: #fff;</string>
    </property>
    <property name="text">
     <string>Professores</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/alunos.png</normaloff>icons/alunos.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTurmas">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>200</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(255, 123, 28);
border-radius: 15px;
background-color: rgb(255, 123, 28);
color: #fff;</string>
    </property>
    <property name="text">
     <string>Turmas</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/turma.png</normaloff>icons/turma.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnServidores">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>200</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(255, 123, 28);
border-radius: 15px;
background-color: rgb(255, 123, 28);
color: #fff;</string>
    </property>
    <property name="text">
     <string>Servidores</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/alunos.png</normaloff>icons/alunos.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>334</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body style=&quot;background-color: #fff;&quot;&gt;&lt;p&gt;Sair&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/exit.png</normaloff>icons/exit.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCargos">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>200</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(255, 123, 28);
border-radius: 15px;
background-color: rgb(255, 123, 28);
color: #fff;</string>
    </property>
    <property name="text">
     <string>Cargos</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/work.png</normaloff>icons/work.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPerfil">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>240</y>
      <width>200</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(255, 123, 28);
border-radius: 15px;
background-color: rgb(255, 123, 28);
color: #fff;</string>
    </property>
    <property name="text">
     <string>Perfil do Adm</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/config.png</normaloff>icons/config.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
